### ISSUE TITLE
Tic Tac Toe Input Output

### ISSUE THREAD

**Issue by Hassan-A** [Hassan-A/mini-projects#1]
I was thinking we could have the output like this:
`    1    2    3`
`1      |     |    `
`    ----------- `
`2      |     |    `
`    ----------- `
`3      |     |    `

Where the player would then input the row/col they want to be positioned

**Comment by aa-repo** [comment]
Yes, sounds good. The player can also enter a number between 1-9 initialy if makes it easier. One things to keep in my mind, is interface will be abstracted way up, so that it will be easy to replace it with a GUI or an API.
